api/nakadi-event-bus-api.yaml

```diff
@@ -1151,6 +1263,36 @@ paths:
 # ################################### #
 
 definitions:
+  Storage:
+      type: object
+      description: |
+        This is a storage backend
+        A real one.
+      properties:
+        id:
+          description: The ID of the storage backend
+          type: string
+        storage_type:
+          description: the type of storage (currently, only Kafka)
+          type: string
+        kafka_configuration:
+          description: configuration settings (currently, only Kafka)
```

Comment:
I thought of using `one_of`, but currently we only have Kafka, so it's a bit useless. Except that it indicates that there may be more storage types in the future